api/src/utils/sum_edits.js

```diff
@@ -24,6 +24,12 @@ const summable = [
  * @param {Object} records
  */
 module.exports = records => {
-  const summableProperties = map(x => Number(x))(reject(isNil, (props(summable, records))))
+  const summableProperties = map(x => {
+    let num = Number(x)
+    if (Number.isNaN(num)) { // control for NaNs in response
+      num = 0
+    }
+    return num
+  })(props(summable, records))
```

Comment:
This function can be shortened, effectively it's `x => ~~x`

Reply:
Thanks @kamicut ! Implemented